pilot/database/database.py

```diff
@@ -120,7 +120,7 @@ def get_all_app_development_steps(app_id, last_step=None, loading_steps_only=Fal
                             (DevelopmentSteps.prompt_path.contains('feature_plan')) |
                             (DevelopmentSteps.prompt_path.contains('feature_summary')))
 
-    return [model_to_dict(dev_step) for dev_step in query]
+    return [model_to_dict(dev_step, recurse=False) for dev_step in query]
```

Comment:
This function (`get_all_app_development_steps`) is used in `get_created_apps_with_steps`, and in Project `dev_steps_to_load`. That one never uses `previous_step` or `user` or `app` as objects, so this change shouldn't have any impact on it.